lib/shopify_import_massager.ex

```diff
@@ -76,18 +65,110 @@ defmodule ShopifyImportMassager do
       substitution_map: substitution_map,
       input_folder: input_folder,
       output_folder: output_folder,
-      filename_parameters: transactions_files_filename_parameters
+      filename_parameters: transactions_files_filename_parameters,
+      file_type: :transactions
     }
     |> substitute()
 
     IO.puts("Processing COMPLETE")
   end
 
+  def extract_substitution_map(
```

Comment:
This is a bit of an adjustment because we want to capture the last order number on the previous file in case the first order number in a file requires a substitution. Thus we switch to `Enum.reduce`